WinRT.Runtime/Projections/IReadOnlyDictionary.net5.cs

```diff
@@ -349,9 +349,15 @@ private ConstantSplittableMap(global::System.Collections.Generic.KeyValuePair<K,
                     return kvArray;
                 }
 
+                public uint Size => (uint)(lastItemIndex - firstItemIndex + 1);
+
+                public global::System.Collections.Generic.IEnumerable<K> Keys => throw new NotImplementedException();
```

Comment:
Can this split map be passed to C# for a Dictionary parameter and be expected to respond to the Keys and Values property?